src/DetailsView/reports/components/automated-check-issue-details-instances.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+import { NamedSFC } from '../../../common/react/named-sfc';
+import { DecoratedAxeNodeResult } from '../../../injected/scanner-utils';
+
+export interface AutomatedChecksIssueDetailsInstancesProps {
+    nodeResults: DecoratedAxeNodeResult[];
```

Comment:
The old report is using `ScanResults` from `src\scanner\iruleresults.d.ts` instead of `DecoratedAxeNodeResult`. Can you have a look at the type and make the appropriate changes? The general idea (if you look at `ReportSectionFactory` on 'src\DetailsView\reports\components\report-sections\report-section-factory.tsx') is to use `SectionProps` (we can talk about the name) to pass all the data the report needs. You may also have a look at how we're already using `SectionProps` in the other sections (header and scan details come to mind)

Reply:
The old report doesn't show `how to fix` information which we need here. I am a little confused as to what property will give me `how to fix` information from the `ScanResults`

Reply:
don't be confused, check the details view -> issue details panel, that's where we are showing it.